athena/athena/athena/endpoints.py

```
@@ -259,168 +303,230 @@ async def wrapper(
+        try:
+            return cls.model_json_schema(by_alias=True)  # v2
+        except AttributeError:
+            return cls.schema(by_alias=True)  # v1
+
+    def _effective_defaults(app_state) -> Dict[str, Any]:
+        cfg = getattr(app_state, "module_config", None)
+        if cfg is not None:
+            return cfg.dict(by_alias=True)
+        try:
+            return cls().dict(by_alias=True)
+        except Exception as exc:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)